rpcs3/rpcs3qt/localized.cpp

```diff
@@ -55,55 +43,31 @@ QString Localized::GetVerboseTimeByMs(qint64 elapsed_ms, bool show_days) const
 	{
 		if (minutes <= 0)
 		{
-			if (seconds == 1)
-			{
-				return tr("%0 second").arg(seconds);
-			}
-			return tr("%0 seconds").arg(seconds);
+			return tr("%n second(s)", "", seconds);
 		}
 
 		if (seconds <= 0)
 		{
-			if (minutes == 1)
-			{
-				return tr("%0 minute").arg(minutes);
-			}
-			return tr("%0 minutes").arg(minutes);
-		}
-		if (minutes == 1 && seconds == 1)
-		{
-			return tr("%0 minute and %1 second").arg(minutes).arg(seconds);
+			return tr("%n minute(s)", "", minutes);
 		}
+
 		if (minutes == 1)
 		{
-			return tr("%0 minute and %1 seconds").arg(minutes).arg(seconds);
-		}
-		if (seconds == 1)
-		{
-			return tr("%0 minutes and %1 second").arg(minutes).arg(seconds);
+			return tr("%0 minute and %n second(s)", "", seconds).arg(minutes);
 		}
-		return tr("%0 minutes and %1 seconds").arg(minutes).arg(seconds);
+		
+        return tr("%0 minutes and %n second(s)", "", seconds).arg(minutes);
```

Comment:
the indentation here seems a bit off

Reply:
Feel free to update that : The "allows edits by maintainers" is enabled no ?